internal/parser/grammar.md

```diff
@@ -99,3 +100,23 @@ To determine which tokens a production accepts, compute `FIRST(X)` for each prod
 1. For a terminal, `FIRST(X) = {X}`.
 2. For a alternation, calculate `FIRST(X)` for each production and form the union.
 3. For concatentation, calculate `FIRST(X)` for the first production. If this set contains the empty set, calculate `FIRST(X)` for the next production. Continue until you hit a production that does not contain the empty set or, if all productions have been evaluated, then the empty set is accepted for this production.
+
+## Operator Precedence
+
+Operator precedence is defined within the grammar itself so a precedence table is not needed. While the table itself is not needed and not used in the parser itself, a table is provided below for human readability with verifying the grammar.
+
+|Precedence|Operator|Description|
+|----------|--------|-----------|
+|1|`a()`|Function call|
+| |`a[]`|Member or index access|
+| |`.`|Member access|
+|2|`*` `/`|Multiplication and division|
+|3|`+` `-`|Addition and subtraction|
+|4|`==` `!=`|Comparison operators|
+| |`<` `<=`| |
+| |`>` `>=`| |
+| |`=~` `!~`| |
+|5|`not`|Unary logical expression|
+|6|`and` `or`|Logical AND and OR|
```

Comment:
Can you align the `|` veritcally, its hard to read in this state?

Reply:
Also can you add this explanation to the SPEC itself, not just this internal grammar doc?